OutOfSchool/OutOfSchool.WebApi/Enums/AchievementTypeLocalization.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OutOfSchool.WebApi.Enums;
+
+public enum AchievementTypeLocalization
```

Comment:
Maybe, it will be better to name it like "LocalizationType" and use everywhere if we need.